app/views/modules/_header.html.erb

```diff
@@ -26,6 +26,7 @@ Unless required by applicable law or agreed to in writing, software distributed
     <div class="col-sm-4">
       <form accept-charset="UTF-8" action="/catalog" id="search-form-header" method="get" role="search" class="form-inline hidden-xs mast-sm-search">
         <div style="display:none"><input name="utf8" type="hidden" value="✓"></div>
+        <input id="search_field" name="search_field" type="hidden" value="all_fields">
```

Comment:
Why is this needed? Shouldn't this be the default of searching all the fields? What happens if a user doesn't pass in this query param? Does search break?